rest/results-tabulation-api/ext/ExtendedElection/ExtendedElectionParliamentaryElection2020/__init__.py

```diff
@@ -293,6 +296,49 @@ def build_election(self, party_candidate_dataset_file=None,
             ]
         )
 
+        tally_sheet_template_pe_22 = Template.create(
+            templateName=PE_22
+        )
+        tally_sheet_template_pe_22_preferences_indicated_for_more_than_three_candidates_row = tally_sheet_template_pe_22.add_row(
```

Comment:
Since invalidVoteCategoryId is taken as a column,is it required to have multiple rows?

Reply:
Fixed.